lib/multi-write.ts

```diff
@@ -209,7 +211,13 @@ async function pipeRegularSourceToDestination(
 			lastPosition = progress.position;
 			onProgress(progress);
 		});
-		sourceStream.pipe(destinationStream);
+		if (!(sourceStream instanceof BlockReadStream) && (destination.destinations.size > 1)) {
+			// Chunk the input stream in a transform if it's not a block read stream, avoiding
+			// chunking it in each destination stream.
+			sourceStream.pipe(new BlockTransformStream(1024 **2)).pipe(destinationStream);
```

Comment:
Nit: space after `**`, and should this maybe use the const `CHUNK_SIZE`?

Reply:
updated